### PR TITLE
Add a SOP to the ghproxy inode alert

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet [hunk 1]
```diff
@@ -64,7 +64,16 @@
               severity: 'critical',
             },
             annotations: {
-              message: '{{ $labels.token_hash }} uses 90% of the available inodes. Check <ghcache_disk_inode_used / ghcache_disk_inode_total * 100 |prometheus>',
+              message: |||
+                {{ $labels.token_hash }} uses 90% of the available inode (<https://grafana-prow-monitoring.apps.ci.l2s4.p1.openshiftapps.com/d/d72fe8d0400b2912e319b1e95d0ab1b3/github-cache?viewPanel=5&orgId=1|dashboard>)
+
+                Resolve by pruning the cache inside the ghproxy pod:
+
+                $ oc --context appci get pod -n ci -l component=ghproxy
```
Comment (hongkailiu) [inline clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet:72]:
```
$ oc --context app.ci -n ci exec $(oc --context app.ci get pod -n ci -l component=ghproxy -o custom-columns=":metadata.name" --no-headers) -- find /cache/data /cache/temp -mtime +1 -type f -delete
```

Comment (petr-muller) [reply clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet:72]: Depends on whether we want people to blindly copy and paste without thinking (nobody will read that condensed thing) or not.

Comment (stevekuznetsov) [reply clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet:72]: I would like to be blind, yes

Comment (stevekuznetsov) [reply clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet:72]: Can we implement pruning in the cache itself? @alvaroaleman

Comment (alvaroaleman) [reply clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet:72]: Yeah, likely (But until we actually do, this SOP is still valid)

Comment (petr-muller) [reply clusters/app.ci/prow-monitoring/mixins/_prometheus/ghproxy_alerts.libsonnet]: Simplified the command, filed https://github.com/kubernetes/test-infra/issues/23407

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**